src/app/chat/views/messages.nim

```diff
@@ -101,6 +101,24 @@ QtObject:
 
     return updatedMessage
 
+  proc userNameOrAlias*(self: MessageView, pubKey: string): string {.slot.} =
+    if self.status.chat.contacts.hasKey(pubKey):
+      return status_ens.userNameOrAlias(self.status.chat.contacts[pubKey])
+    generateAlias(pubKey)
+
+  proc replacePubKeysWithMentions(self: MessageView, message: string, contacts: Table[string, Profile]): string =
```

Comment:
Don't we already have this kind of function somewhere, for example when we render message? That would probably be better to reuse it